channeldb/payment_control.go

```diff
@@ -458,7 +509,9 @@ func (p *PaymentControl) FetchInFlightPayments() ([]*InFlightPayment, error) {
 				return nil
 			}
 
-			var inFlight = &InFlightPayment{}
+			var inFlight = &InFlightPayment{
```

Comment:
This can just be:
```go
var inFlight &InFlightPayment
```

Then pass in the pointer at the append at the bottom of the transaction.

Reply:
I don't think this still applies. There is an initializer for the htlc slice too now.